pkg/api/router.go

```diff
@@ -28,6 +28,14 @@ func (s *server) setupRouting() {
 		fmt.Fprintln(w, "User-agent: *\nDisallow: /")
 	})
 
+	router.Handle("/bzz-file", jsonhttp.MethodHandler{
+		"POST": http.HandlerFunc(s.bzzFileUploadHandler),
+	})
+
+	router.Handle("/bzz-file/{addr}", jsonhttp.MethodHandler{
+		"GET": http.HandlerFunc(s.bzzFileDownloadHandler),
+	})
+
```

Comment:
Could you update this branch to the latest master and use the handle function from there as:

```
	handle(router, "/files", jsonhttp.MethodHandler{
		"POST": http.HandlerFunc(s.fileUploadHandler),
	})
	handle(router, "/files/{address}", jsonhttp.MethodHandler{
		"GET": http.HandlerFunc(s.fileGetHandler),
	})
```